src/transformers/models/altclip/processing_altclip.py

```diff
@@ -35,22 +58,16 @@ class AltCLIPProcessor(ProcessorMixin):
             The image processor is a required input.
         tokenizer ([`XLMRobertaTokenizerFast`], *optional*):
             The tokenizer is a required input.
+        feature_extractor ([`CLIPFeatureExtractor`], *optional*):
+            The feature extractor is a deprecated input.
```

Comment:
Given we don't advertise `audio` is an input when calling the processor, do we want to add `feature_extractor` here to the docstring?

Reply:
Was hesitant on that - wdyt about not advertising unused modalities, rather? I wanted to have the same signature everywhere but that might be overkill

Reply:
oh, no I understand the issue here @amyeroberts , in this case `feature_extractor` is unfortunately a backwards compatible name for the image processor. So we cannot remove it.

Reply:
Yes, I think we need to be able to accept feature_extractors as an input, but I don't think they should be added to the docstring as it's confusing. This probably falls under the same scope for @qubvel's deprecation decorators again

Reply:
agreed! done!